views/courses/components/topics.hbs

```diff
@@ -34,7 +34,8 @@
 				{{#unless this.hidden}}
 				<!-- TOPIC IS NOT HIDDEN, teacher and student has access -->
 				<div class="card card-topic locationlink" data-loclink="{{this.url}}" {{#userHasPermission "COURSE_EDIT"}}
```

Comment:
if we have a role of button and tabindex then it makes sense to change it to a button

Reply:
This would require deeper rework of scripts and style... Do you really think its required?

Reply:
This is kind of an anti-pattern... can't we have a class that will style it properly?

Reply:
done :)